tests/sources/unix.elpi

```diff
@@ -10,9 +10,10 @@ pred works1.
 works1 :-
   std.assert-ok! (open-process ["cat"] _ P) "works1",
   P = (process In Out _Err),
-  output In "foo",
+  output In "foo\n",
```

Comment:
Why the newline?

Reply:
blind attempt. Now it is the `works1` test that fails, and `input_line` should look for a EOL.
So I don't know why it worked before (even on recent versions of ocaml).